spec/model/post_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
- [ ] Please ensure that any custom methods in the models have a corresponding unit test, So make sure you add a unit test for the `recent_comments` method in your post model :blush: